cypress/support/commands.js

```diff
@@ -24,47 +24,56 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
+// Логін користувача через API
+// Реєстрація користувача через API
+// Реєстрація користувача через API
 
+Cypress.Commands.add('registerUser', (email, username, password) => {
+  cy.request({
+    method: 'POST',
+    url: '/api/users',
+    body: {
+      user: {
+        email,
+        username,
+        password
+      }
+    },
+    failOnStatusCode: false // Додано, щоб уникнути помилки через вже зайнятий email
```

Comment:
It's a good practice to handle potential errors in your tests. Here, you're doing this by setting 'failOnStatusCode' to false.